src/effects/chapter_change.c

```diff
@@ -295,6 +264,13 @@ void chapter_change_appendGfx(void* effect) {
         case 16:
         case 17:
         case 18:
+#if VERSION_PAL
+            if (gCurrentLanguage == LANGUAGE_DE) {
+                ptr0 = D_E010E838_b;
+                ptr1 = D_E010E838_c;
```

Comment:
Maybe we can rename D_E010E838_b and D_E010E838_c to D_E010E838_de D_E010E838_de? Or is it better to wait until we fully know what it is

Reply:
I'd prefer to wait